CHANGELOG.rst

```diff
@@ -17,25 +17,29 @@ Change Log
 
 Changed
 =======
-- added new catalog-info field ``openedx.org/release``
-- removed openedx.yaml file
+
+- Added new catalog-info field ``openedx.org/release``
+- Removed openedx.yaml file
```

Comment:
I'm not sure what style guide you're following (if any), but it seems like you added code formatting to file names in other places. Do you want to add it here?

i.e. 
```suggestion
- Removed ``openedx.yaml`` file
```

Reply:
Nice catch, @pdpinch. I'll fix it.

Reply:
Nice catch, @pdpinch. I'll fix it.